tests/test_tasks_downstream.py

```diff
@@ -177,75 +177,48 @@ def test_install(
                 invoke("img-build")
                 invoke("git-aggregate")
                 invoke("resetdb")
-            # Install "purchase"
-            ret_code, stdout, stderr = invoke.run(("install", "-m", "purchase"))
-            assert "Executing odoo --stop-after-init --init purchase" in stderr
-            assert _install_status("purchase") == "installed"
-            assert _install_status("sale") == "uninstalled"
-            # Change to "sale" subfolder and install
+            # Install "mail"
+            stdout = invoke("install", "-m", "mail")
+            assert "Executing odoo --stop-after-init --init mail" in stdout
+            assert _install_status("mail") == "installed"
+            assert _install_status("utm") == "uninstalled"
+            assert _install_status("note") == "uninstalled"
+            # Change to "utm" subfolder and install
             with local.cwd(
-                tmp_path / "odoo" / "custom" / "src" / "odoo" / "addons" / "sale"
+                tmp_path / "odoo" / "custom" / "src" / "odoo" / "addons" / "utm"
             ):
-                # Install "sale"
-                ret_code, stdout, stderr = invoke.run("install")
-                assert "Executing odoo --stop-after-init --init sale" in stderr
-                assert _install_status("purchase") == "installed"
-                assert _install_status("sale") == "installed"
-    finally:
-        # Imagine the user is in the odoo subrepo for this command
-        with local.cwd(tmp_path / "odoo" / "custom" / "src" / "odoo"):
-            invoke("stop", "--purge")
-
-
-@pytest.mark.sequential
-def test_test(
-    cloned_template: Path,
-    docker: LocalCommand,
-    supported_odoo_version: float,
-    tmp_path: Path,
-):
-    """Test the test task.
-
-    On this test flow, other downsream tasks are also tested:
-
-    - img-build
-    - git-aggregate
-    - stop --purge
-    """
-    try:
-        with local.cwd(tmp_path):
-            copy(
-                src_path=str(cloned_template),
-                vcs_ref="HEAD",
-                force=True,
-                data={"odoo_version": supported_odoo_version},
+                # Install "utm" based on current folder
+                stdout = invoke("install")
+                assert "Executing odoo --stop-after-init --init utm" in stdout
+                assert _install_status("mail") == "installed"
+                assert _install_status("utm") == "installed"
+                assert _install_status("note") == "uninstalled"
+            # Test "note" simple call in init mode (default)
+            stdout = invoke("test", "-m", "note", "--mode", "init", retcode=None)
+            assert (
+                "Executing odoo --test-enable --stop-after-init --workers=0 -i note"
+                in stdout
             )
```

Comment:
I think this'd be important:

```suggestion
            )
            assert "devel odoo.modules.loading: All post-tested" in stdout
```